highway/server.go

```diff
@@ -96,28 +100,55 @@ func NewHighway(ctx context.Context, opts ...hn.Option) (*HighwayServer, error)
 	// purges expired items every 10 minutes
 	c := cache.New(5*time.Minute, 10*time.Minute)
 
+	// TODO work with Nick on what exact approach to do on this
```

Comment:
I think it will crash without this configuration directory, we should probably uncomment this for now

Reply:
It will work without this for ephemeral nodes which is what we are running currently. I will work with Nick at some point to work out the kinks with default nodes. I think this process here would be better done in a dockerfile but I'm leaving this here till that is taken care of.